### DISCUSSION TITLE
Threshold dependent interventions

### DISCUSSION THREAD

**Discussion by pratikunterwegs** [epiverse-trace/epidemics#234] [Ideas]
This issue was flagged by @BlackEdder: it would be useful to allow for a class of interventions that are activated when a threshold is crossed, and that are deactivated at another threshold. Examples include NPIs to reduce contacts when the number of daily hospitalisations exceeds a threshold.

**Comment by rozeggo** [comment]
I think this would be a really useful addition. However it's possibly quite infection-specific on whether it would be needed, e.g. it isnt needed for ebola, so I am not sure it's high priority to include in the short term. Probably a medium-term priority.
